README.md

```diff
@@ -11,10 +11,11 @@ Demo: http://collabit.herokuapp.com
 ### *nix Install
 
 1. `brew/apt-get install node npm`
-3. `git clone https://github.com/ehaughee/collabit.git`
-4. `cd collabit`
-5. `npm install`
-7. `node app.js`
+2. `git clone https://github.com/ehaughee/collabit.git`
+3. `cd collabit`
+4. `npm install`
+5. Optionally, if you are using VSCode and want intellisense: 'npm run devsetup'
```

Comment:
Change the apostrophes in 'npm run devsetup' to backticks so it will look like this: `npm run devsetup`.